tests/ui/parser/integer-literal-start-ident.stderr

```diff
@@ -0,0 +1,14 @@
+error: expected identifier, found `1main`
+  --> $DIR/integer-literal-start-ident.rs:1:4
+   |
+LL | fn 1main() {}
+   |    ^^^^^ expected identifier
+   |
+help: identifiers cannot start with a number
+  --> $DIR/integer-literal-start-ident.rs:1:4
+   |
+LL | fn 1main() {}
+   |    ^^^^^
```

Comment:
the `help` is always redundant with its span. Maybe just make it an unspanned note?